aggregator/src/app/BundleTable.ts

```diff
@@ -24,20 +25,27 @@ import nil from "../helpers/nil.ts";
  */
 type RawRow = {
   id: number;
+  status: string;
   hash: string;
   bundle: string;
   eligibleAfter: string;
   nextEligibilityDelay: string;
   submitError: string | null;
+  receipt: string | null;
 };
 
+const BundleStatuses = ["pending", "confirmed", "failed"] as const;
```

Comment:
Is it possible for this to be an enum? Would that make parsing the type easier? More of an open question than a suggestion

Reply:
Just realised I didn't respond to these - thanks for the detailed answers!

Ok gotcha, sounds good to me.